Singular/dyn_modules/cohomo/cohomo.cc

```diff
@@ -4134,7 +4135,7 @@ std::vector<std::vector<int> > bsubsets_1(poly b)
   std::vector<std::vector<int> > bset;
   for(int i=0;i<bvs.size();i++)
   {
-    for(int j=0;j<bvs.size(), j!=i; j++)
+    for(int j=0;j!=i; j++)
```

Comment:
The new code is equivalent to the old, and also seems sensible to me, but somebody who knows this code should verify this is what was the intention (and that not e.g. `&&` instead of `,` was intended)